Sources/File.swift

```diff
@@ -188,14 +188,10 @@ extension File {
         return filedetach(file)
     }
 
-    public func close() -> Bool {
-        if closed {
-            return false
-        }
-
+    public func close() throws {
+        guard !closed else { throw ClosableError.alreadyClosed }
```

Comment:
@dreymonde agree, we should move send/receive to FileStream and make File simple with read/write functions